legacy/dockerregistry/inventory.go

```diff
@@ -1950,6 +1950,40 @@ func SplitRegistryImagePath(
 		fmt.Errorf("could not determine registry name for '%v'", registryImagePath)
 }
 
+func (sc *SyncContext) ValidateEdges(edges map[PromotionEdge]interface{}) error {
+	errors := []error{}
```

Comment:
When you're validating some data structure, it helps to see the entire thing to check for all possible errors, instead of just returning the very first thing you see. This is mirrored in real life, such as a physical exam at a doctor's office --- the doctor will not stop checking things as soon as they see the first thing that's abnormal (instead they will go through all the tests that they have for you).

Reply:
Understood. This was more of a nit.